plasmapy/mathematics/mathematics.py

```diff
@@ -110,7 +111,7 @@ def plasma_dispersion_func_deriv(zeta):
 
     See also
     --------
-    plasma_dispersion_func
+    plasma_dispersion_func_deriv
```

Comment:
Circular reference, it was correct before.

Reply:
good catch